pages/about/index.vue

```diff
@@ -11,69 +11,75 @@
     <breadcrumb :breadcrumb="breadcrumb" title="About" />
     <page-hero class="py-24" v-if="heroCopy">
       <h1>{{ pageTitle }}</h1>
-      <div v-html="parseMarkdown(heroCopy)" />
+      <div v-html="parseMarkdown(heroCopy)"></div>
     </page-hero>
     <div class="page-wrap container">
-      <h3>What are we doing here at Epilepsy.Science?</h3>
+      <div class="epilepsy-about">
+        <h3>What are we doing here at Epilepsy.Science?</h3>
 
-      <p>
-        Epilepsy.Science is a new cloud-based platform for managing, analyzing,
-        publishing, and sharing scientific datasets to accelerate epilepsy
-        research. Led by PennSieve and the Brain Data Science Platform (BDSP)
-        and with support from the AWS Open Data Sponsorship Program, we are
-        creating an unparalleled open data resource for the epilepsy community.
-      </p>
+        <p>
+          Epilepsy.Science is a new cloud-based platform for managing, analyzing,
+          publishing, and sharing scientific datasets to accelerate epilepsy
+          research. Led by PennSieve and the Brain Data Science Platform (BDSP)
```

Comment:
We don't usually capitalize PennSieve like that. I think this should be 
`Pennsieve`